src/backend/allocator/format.rs

```diff
@@ -94,218 +107,238 @@ macro_rules! format_tables {
 
 format_tables! {
     // 8-bit bpp Red
-    R8 { alpha: false, bpp: 8 },
+    R8 { alpha: false, bpp: 8, depth: 8 },
 
     // TODO: Update drm-fourcc
     // 16-bit bpp Red with padding (x:R)
     // R10 { bpp: 16 },
     // R12 { bpp: 16 },
```

Comment:
Let's add the depth to these also, we do need to update drm-fourcc to get these variants though

Reply:
done